README.md

```diff
@@ -0,0 +1,49 @@
+# aleph-client
+
+Python Client for the [aleph.im network](https://www.aleph.im), next generation network of
+decentralized big data applications. Developement follows the [Aleph
+Whitepaper](https://github.com/aleph-im/aleph-whitepaper).
+
+## Documentation
+
+Documentation can be found on https://docs.aleph.im/tools/aleph-client/
+
+## Requirements
+
+### Linux
+
+Some cryptographic functionalities use curve secp256k1 and require
+installing [libsecp256k1](https://github.com/bitcoin-core/secp256k1).
+
+> apt-get install -y python3-pip libsecp256k1-dev
+
+### macOs
+
+> brew tap cuber/homebrew-libsecp256k1
+> brew install libsecp256k1
+
+## Installation
+
+Using pip and [PyPI](https://pypi.org/project/aleph-client/):
+
+> pip install aleph-client
+
+## Installation for development
```

Comment:
Shouldn't we also add instructions regarding `hatch` here?

I see some of them for the python SDK https://github.com/aleph-im/aleph-sdk-python?tab=readme-ov-file#installation-for-development

Reply:
Good point. Added via commit https://github.com/aleph-im/aleph-client/pull/213/commits/e97df3b8e0401f0f16dc6daa1ef10787d156c696